### DISCUSSION TITLE
Github account

### DISCUSSION THREAD

**Discussion by okekingscodes** [community/community#19609] [Other Feature Feedback, Questions, & Ideas]
Fellow developers, is it generally acceptable to post your github account on social media? 

For me I feel like, your github account is like your profile so sharing it on social media is like sharing your open source codes.

I maybe wrong though.

I need better understanding please.

**Comment by melvincwng** [comment]
@okekingscodes, honestly I think it's up to personal preference.

At least, based on my experience, usually my fellow developer friends do not share anything with regards to their GitHub on conventional social media platforms like Facebook, Twitter, Instagram etc.

But, in the case of professional social network platforms like LinkedIn, definitely I believe you should share your GitHub profile over there. So that potential recruiters can look at your source code and determine if you are a good fit for the role (assuming they are hiring).

Don't fret too much on this, like I said earlier, it's just a matter of preference.

Just my two cents though :)

Hope this helps.
